examples/qiskit/0_Getting_Started.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>* I think if you ran this within the notebook, it would effectively change the region for the notebook interactions with the service through the BDK as well.</p><p>* A lot of the information here is duplicated from our dev guide. There's a possibility this might not be updated when, for example, we launch TN1 in a new region. Would it be possible to reference the dev guide instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I agree with the second point. For the first point, since this is a code snippet in a markdown cell and not a code cell, the code cannot execute and therefore will not change the region.</p>

Reply:
<p>Basically, if I want to demo anything about TN1, it means that I need to switch regions, and if you're a Qiskit user, you'll have no idea what regions are or why you need to care about them, and IMO you probably won't want to have to go all the way to the dev guide to read up on something; you'll want it to just work. I don't know of an optimal solution here.</p>